lib/spack/docs/packaging_guide.rst

```diff
@@ -3602,21 +3602,23 @@ In this case the package is passed as the second argument, and ``self`` is the b
 
 .. _multiple_build_systems:
 
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
 Multiple build systems
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
```

Comment:
If we want to move this up one level, we need to move it below "Mixin base classes" (which are not a subsection of multiple build systems).